apps/design/backend/src/worker/generate_election_package.ts

```diff
@@ -0,0 +1,51 @@
+import { createWriteStream } from 'fs';
+import JsZip from 'jszip';
+import path from 'path';
+import { pipeline } from 'stream/promises';
+import { layOutAllBallotStyles } from '@votingworks/hmpb-layout';
+import {
+  BallotType,
+  ElectionPackageFileName,
+  getDisplayElectionHash,
+  Id,
+} from '@votingworks/types';
+
+import { PORT } from '../globals';
+import { WorkerContext } from './context';
+
+export async function generateElectionPackage(
```

Comment:
No real changes in this commit, just a copy-paste out of one file and into another